indico_toolkit/association/positioning.py

```diff
@@ -42,6 +42,33 @@ def positioned_above(
             is_above = False
         return is_above
 
+    def positioned_above_overlap(
+            self, above_pos: dict, below_pos: dict, min_overlap_percent: float = None
+        ) -> bool:
+        """
+        Check if the location of one box is above another and if the lower box's overlap is at least the given percentage.
+        Args:
+            above_pos (dict): the position expected to be above
+            below_pos (dict): the position expected to be below
+            min_overlap_percent (float, optional): the minimum amount of overlap needed. Defaults to None.
+
+        Returns:
+            bool: True is above_pos is above below_pos and below_pos' amount of overlap is at least min_overlap_percent
+        """
+        is_above = False
+        is_min_overlap = True
+        if below_pos["page_num"] < above_pos["page_num"]:
+            raise ToolkitInputError(
+                "Predictions are not on the same page!"
+            )
+        if self.xaxis_overlap(above_pos, below_pos):
+            is_above = True
+            horizontal_overlap_distance = abs(max(above_pos["bbLeft"], below_pos["bbLeft"]) - min(above_pos["bbRight"], below_pos["bbRight"]))
+            position_width = abs(below_pos["bbLeft"] - below_pos["bbRight"])
+            if horizontal_overlap_distance / position_width < min_overlap_percent:
```

Comment:
does this need an "and" conditional on "min_overlap_percent"? i.e. if this condition and min_overlap_percent is not None